test/perf/tests/styled-system-spaces/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
As the current import will stop working in the future versions of React.

```suggestion
import * as React from 'react';
```